torch2jax/__init__.py

```diff
@@ -915,6 +984,18 @@ def relu(x, inplace=False):
     return Torchish(jax.nn.relu(_v(x)))
 
 
+@implements(torch.nn.functional.leaky_relu, Torchishify_output=False)
+def leaky_relu(x, negative_slope=0.01, inplace=False):
```

Comment:
```suggestion
def leaky_relu(input, negative_slope=0.01, inplace=False):
```

so that `leaky_relu(input=foo)` works, consistent with torch's argument names